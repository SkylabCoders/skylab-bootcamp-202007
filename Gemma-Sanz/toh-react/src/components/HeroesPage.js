import React from 'react';
import { Link } from 'react-router-dom';
import HeroList from "./HeroList";

function HeroesPage() {
    return (
        <>
            <p>
                <Link to="/hero"> + Add Hero </Link>
            </p>
            <HeroList />
        </>
    )
}