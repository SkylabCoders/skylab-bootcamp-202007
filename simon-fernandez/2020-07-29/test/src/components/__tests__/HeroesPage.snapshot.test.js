import renderer from 'react-test-renderer';
import HeroesPage from '../HeroesPage';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';

describe('App Snapshot', () => {
	const tree = renderer.create(
		<Router>
			<HeroesPage />
		</Router>
	);

	it('should match snapshot', () => {
		expect(tree.toJSON()).toMatchSnapshot();
	});
});
