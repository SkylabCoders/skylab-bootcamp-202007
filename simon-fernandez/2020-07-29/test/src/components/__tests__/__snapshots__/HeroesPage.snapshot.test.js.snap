// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot should match snapshot 1`] = `
Array [
  <p>
    <a
      href="/hero"
      onClick={[Function]}
    >
      + Add Hero
    </a>
  </p>,
  <ul />,
]
`;
